apps/application-system/api/src/app/modules/application/utils/validationUtils.ts

```diff
@@ -46,15 +46,15 @@ export async function validateIncomingAnswers(
   application: Application,
   newAnswers: FormValue | undefined,
   isStrict = true,
+  nationalId = '',
```

Comment:
Same arguments here, will this then not become a required parameter that is not initialised?